### ISSUE TITLE
Revert a release to a previous version

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1672]

**Comment by colinmurphy01** [comment]
@greg-does-weather revisiting to see if we should consider prioritizing this over the next couple sprints?

**Comment by greg-does-weather** [comment]
It doesn't seem urgent to me. @jamestranovich-noaa What do you think? I wonder if it makes the most sense to make this change when we migrate to Gitlab since we'll have to rebuild our CI/CD anyway.

**Comment by jamestranovich-noaa** [comment]
I agree. If we can wait on this one, let's do this when we migrate to Gitlab. Otherwise we'll be duplicating work (which can be fine if we need this feature now and not later).
